tests/features/eulogin/eulogin.feature

```diff
@@ -134,13 +134,14 @@ Feature: Log in through EU Login
       | Username | E-mail     | Password           | First name | Last name | Local username |
       | jb007    | 007@mi6.eu | shaken_not_stirred | James      | Bond      | jb007_local    |
 
-    # Try to login using the Drupal login form.
+    # Try to login using the Drupal login form. The user should be redirected to
+    # EU Login.
     Given I go to "/user/login"
-    And I fill in "Email or username" with "jb007_local"
-    And I fill in "Password" with "123"
-    And I wait for the honeypot time limit to pass
-    When I press "Sign in"
-    Then I should see the error message "Please sign in with your EU Login account."
+    Then I should see the heading "Sign in to continue"
+    # Also the "user" page which in a normal Drupal installation redirects to
+    # the Drupal login form should now redirect to EU Login.
+    When I visit "user"
+    Then I should see the heading "Sign in to continue"
```

Comment:
Is this assertion correct? We've been decided that there's no redirect but a page w/o the form and still wit the warning banner.

Reply:
This is temporary and will change in ISAICP-5760 which is going to enable the user login again and show the warning as written in the ticket description. The user login has been completely disabled in this PR to be 100% sure that no tests can sneakily log in a user through the old form.

I will add a `Todo` here so that it will not be forgotten.